resource_obmcs_objectstorage_object.go

```diff
@@ -8,10 +8,69 @@ import (
 	"github.com/hashicorp/terraform/helper/schema"
 	"github.com/oracle/bmcs-go-sdk"
 
+	"crypto/md5"
+	"encoding/hex"
+	"strings"
+
 	"github.com/oracle/terraform-provider-oci/crud"
 )
 
 func ObjectResource() *schema.Resource {
+	var objectSchema = map[string]*schema.Schema{
+		"namespace": {
+			Type:     schema.TypeString,
+			Required: true,
+			Computed: false,
+		},
+		"bucket": {
+			Type:     schema.TypeString,
+			Required: true,
+			Computed: false,
+		},
+		"object": {
+			Type:     schema.TypeString,
+			Required: true,
+			Computed: false,
+		},
+		"content": {
+			Type:     schema.TypeString,
+			Optional: true,
+			StateFunc: func(body interface{}) string {
+				v := body.(string)
+				if v == "" {
+					return ""
+				}
+				h := md5.Sum([]byte(strings.TrimSpace(v)))
```

Comment:
Should we trim space? I would think that the string is data that should stay intact.

Reply:
agree